### PR TITLE
Update lifecycle rule

### PR DESCRIPTION
**Pull request by sakomws**
# What
Update lifecycle rule
Update AWS provider to 4

# Why
It was conflicting with existing rule

# Testing
- Rerun the tf plan and make sure no changes

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -126,6 +110,22 @@ resource "aws_s3_bucket_server_side_encryption_configuration" "cdn_bucket" {
   }
 }
 
+resource "aws_s3_bucket_lifecycle_configuration" "default" {
+  count = var.enable_origin_bucket_intelligent_tiering_lifecycle_rule != [] ? 1 : 0
```
Comment (albertrdixon) [inline main.tf]: Isn't this a bool?

**Review by albertrdixon — CHANGES_REQUESTED** [review]
Need to update the condition

File: main.tf (hunk 1 above)
Comment (sakomws) [reply main.tf]: snap, updated. thank uuu

Comment (sakomws) [reply main.tf]: Dear @albertrdixon i addressed your comments, please kindly find a time in your over booked schedule to rereview. Thank you, your truly work mate Sako

**Review by albertrdixon — APPROVED** [review]

**Pull request merged**